src/main/java/org/sagebionetworks/bridge/exporter/scheduler/BridgeExporterScheduler.java

```diff
@@ -71,24 +73,26 @@ public void schedule(String schedulerName) throws IOException {
         String scheduleTypeStr = schedulerConfig.getString("scheduleType");
         if (Strings.isNullOrEmpty(scheduleTypeStr)) {
             // Defaults to DAILY
-            scheduleType = ScheduleType.DAILY;
+            scheduleType = DAILY;
         } else {
             try {
                 scheduleType = ScheduleType.valueOf(scheduleTypeStr);
             } catch (IllegalArgumentException ex) {
                 // log error, default to DAILY
                 System.err.println("Invalid schedule type: " + scheduleTypeStr);
-                scheduleType = ScheduleType.DAILY;
+                scheduleType = DAILY;
             }
         }
 
         // insert date and tag
         switch (scheduleType) {
             case DAILY: {
                 // Each day, we export the previous day's data.
-                String yesterdaysDateString = LocalDate.now(timeZone).minusDays(1).toString();
-                String tag = "[scheduler=" + schedulerName + ";date=" + yesterdaysDateString + "]";
-                requestNode.put("date", yesterdaysDateString);
+                DateTime endDateTime = DateTime.now(timeZone).withHourOfDay(0).withMinuteOfHour(0).withSecondOfMinute(0).withMillisOfSecond(0).minusMillis(1);
```

Comment:
For daily, we use end of the previous day. For hourly, we use start of the current hour. This is inconsistent. We should make these consistent.

Also we should use withTimeAtStartOfDay().